include/libtorrent/magnet_uri.hpp

```diff
@@ -83,6 +83,7 @@ namespace libtorrent {
 	TORRENT_EXPORT add_torrent_params parse_magnet_uri(string_view uri, error_code& ec);
 	TORRENT_EXPORT add_torrent_params parse_magnet_uri(string_view uri);
 	TORRENT_EXPORT void parse_magnet_uri(string_view uri, add_torrent_params& p, error_code& ec);
+	TORRENT_EXPORT void parse_magnet_uri(string_view uri, add_torrent_params& p, error_code& ec, string_view ca_certificate);
```

Comment:
this looks like it has limited value. You could just set the member on the `add_torrent_params` later anyway